insights/specs/insights_archive.py

```diff
@@ -173,6 +173,7 @@ class InsightsArchiveSpecs(Specs):
     modinfo_veth = simple_file("insights_commands/modinfo_veth")
     modinfo_vmxnet3 = simple_file("insights_commands/modinfo_vmxnet3")
     mokutil_sbstate = simple_file("insights_commands/mokutil_--sb-state")
+    mpirun_version = simple_file("insights_commands/mpirun")
```

Comment:
@mohitkumarrh This should be `insights_commands/usr.local.bin.mpich_--version`


```
$ mangle "/usr/local/bin/mpich --version"
insights_commands/usr.local.bin.mpich_--version
```

Reply:
@psachin - Thanks for the review, I have updated the MR.